.travis.yml

```diff
@@ -13,32 +13,29 @@ addons:
 
 cache:
   pip: true
-  ccache: true
   directories:
     - /home/travis/virtualenv
     - $HOME/.pip-cache/
-before_cache:
-    - rm -f .tox/py34-django19/log/*.log
-    - rm -f .tox/py35-django19/log/*.log
-    - rm -f .cache/pip/log/*.log
-
+    - $HOME/.tox/
+    - .tox
 env:
   - ES_VERSION=2.3.1 ES_DOWNLOAD_URL=https://download.elastic.co/elasticsearch/release/org/elasticsearch/distribution/tar/elasticsearch/${ES_VERSION}/elasticsearch-${ES_VERSION}.tar.gz
 
 install:
+  - pip install tox coveralls tox-travis
   # Install ES
   - wget -nc ${ES_DOWNLOAD_URL}
   - tar -xzf elasticsearch-${ES_VERSION}.tar.gz
   - ./elasticsearch-${ES_VERSION}/bin/elasticsearch &
   # Make sure pip is at the latest version:
   - pip install -U pip
   # Now install the rest of the required Python packages:
-  - CFLAGS="-O0" pip install -r requirements.txt
-  - pip install python-coveralls
```

Comment:
When this build completed, it didn't upload a coverage report to coveralls.

Maybe a few lines above where you've added `pip install tox coveralls tox-travis` that should read `pip install tox python-coveralls tox-travis`? Possibly its something else?

Whatever it is, coverage reports should be working before merge.

Reply:
I reckon this will be fixed in 
5a62e95, but let's wait for these tests to pass first.